src/containers/Review/ReviewHome.tsx

```diff
@@ -17,31 +18,43 @@ const ReviewHome: React.FC<ReviewHomeProps> = ({ assignments, structure }) => {
     firstRunValidation: false,
     shouldCalculateProgress: false,
   })
+  const {
+    userState: { currentUser },
+  } = useUserState()
+  // default should really be false, but for testing this is much quicker
+  const [viewAllAssignments, setViewAllAssignment] = useState<boolean>(true)
 
   if (error) return <Message error title={strings.ERROR_APPLICATION_OVERVIEW} list={[error]} />
 
   if (!fullApplicationStructure) return <Loading />
 
-  // i think we would group by stage, and current stage would be the expanded one, for now using latest stage
-  const currentStageAssignments = assignments.filter(
-    (assignment) => assignment.stage.id === fullApplicationStructure.info.current?.stage.id
-  )
-
   return (
     <>
-      {fullApplicationStructure.sortedSections?.map(({ details: { id, title } }) => (
+      <Segment>
+        <Checkbox
+          toggle
+          label="Show All"
```

Comment:
I think this will change, was just an experimental one (look like it would change based on Debbies Designs, so should be ok to keep like this (until it's replaced)

Reply:
All good